pkg/controller/infinispan/infinispan_controller_test.go

```diff
@@ -42,20 +42,20 @@ type mockCluster struct{}
 // GetClusterMembers returns a fake cluster view, produced returning the substring
 // after the - char of the `name` arg. If the substring doesn't start with View and error
 // will be also returned
-func (m mockCluster) GetClusterMembers(ns, name string) (string, error) {
+func (m mockCluster) GetClusterMembers(ns, name, secretName string) ([]string, error) {
```

Comment:
What's the point of `secretName` if it's not used in the function?

Reply:
This file contains a test that mocks Kubernetes, it's a unit test. Hence, secretName is unused, but it's used in the production version, where the secret is used to query the operator password and do the curl to obtain the cluster health.

Reply:
Btw, I've changed this now to use simply use a mock function when testing.